invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_records_search/components.js

```diff
@@ -59,7 +59,13 @@ const DeleteDraftButton = (props) => {
 
   return (
     <>
-      <Button compact size="small" floated="right" color="red" onClick={handleOpen}>
+      <Button
```

Comment:
@lnielsen @ntarocco should we enforce black or something.... I'm seeing changes like this all the time which are style only but change the git history....